### DISCUSSION TITLE
I really want to add

### DISCUSSION THREAD

**Discussion by terrorsl** [terrorsl/sMQTTBroker#36] [Ideas]
Describe what you miss and would like to see in future releases.

**Comment by xenoeng** [comment]
More simultaneous clients please! :-)

**Reply by terrorsl** [reply]
hi, can you elaborate?

**Comment by tttonyyy** [comment]
Would love to see QoS 2 :)

**Reply by terrorsl** [reply]
added, but does not work exactly according to the standard

**Comment by Tinus1988** [comment]
Hi, as a broker I would love to be able to get the IP address of the publishing clients
